### PR TITLE
release: v1.9.0

### PR DESCRIPTION
**Pull request by Conaclos**
## Summary

1. [x] Add a [changelog](./CHANGELOG.md) entry for every Pull Request that lacks one.
   You can filter [merged PRs that don't update the changelog](https://github.com/biomejs/biome/pulls?q=is%3Apr+is%3Amerged+-label%3AA-Changelog).
   Read our [guidelines for editing the changelog](#changelog).

1. [x] Based on the [changelog](./CHANGELOG.md), determine which version number to use.
   See our [versioning guide](https://biomejs.dev/internals/versioning/) for more details.

1. [x] Rename `Unreleased` to `<version> (iso-date)` in the [changelog](./CHANGELOG.md).

1. [x] Update `version` in [Biome's `package.json`](./packages/@biomejs/biome/package.json) if applicable.

1. [ ] **Update to the same `version` in all crates** if you publish crates. (`Cargo.toml` and `crates/**/Cargo.toml`)

1. [x] Linter rules have a `version` metadata directly defined in their implementation.
   This field is set to `next` for newly created rules.
   This field must be updated to the new version.

1. [x] Merge website PRs labeled "Merge on release".

1. [ ] Update the website with the new version number:
   `BIOME_VERSION=<version> just gen-web`.
   This will also copy the configuration schema in the right place.

1. [ ] Once the PR is merged, the CI will trigger the `Release: *` workflows. Once these workflows finish compiling the final artefact, **they need to be approved manually**.

### CODE REVIEW

**Comment by Conaclos** [comment]
@togami2864 Which CSS rules do you think are ready to be promoted to the stable group?

**Comment by arendjr** [comment]
@Conaclos We should also not forget to merge website PRs labeled "Merge on release".

**Comment by Conaclos** [comment]
> We should also not forget to merge website PRs labeled "Merge on release".

Thanks for the reminder! I added a bullet point.

**Comment by ematipico** [comment]
A couple of other things for the website:
- update the language support page with graphql 
- documentation for assists (or I can do it when I come back)
- documentation for VSCode v3
- document the guide to explain that with `--verbose` is now possible to see all fixed files

**Comment by togami2864** [comment]
@Conaclos 
I think they would be good to be promoted

### From stylelint-recommended
- noDuplicateAtImportRules
- noDuplicateFontNames
- noDuplicateSelectorsKeyframeBlock
- noEmptyBlock
- noImportantInKeyframe
- noInvalidDirectionInLinearGradient
- noInvalidPositionAtImportRule
- noIrregularWhitespace
- noShorthandPropertyOverrides
- noUnknownFunction
- noUnknownMediaFeatureName
- noUnknownProperty
- noUnknownUnit
- noUnmatchableAnbSelector
- useConsistentGridAreas
- useGenericFontNames

### Other
- noValueAtRule


| stylelint                                                                                                                             | biome                             | note                                                                                     |
|---------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------|------------------------------------------------------------------------------------------|
| [no-duplicate-at-import-rules](https://stylelint.io/user-guide/rules/no-duplicate-at-import-rules/)                                   | noDuplicateAtImportRules          |                                                                                          |
| [font-family-no-duplicate-names](https://stylelint.io/user-guide/rules/font-family-no-duplicate-names/)| noDuplicateFontNames | |
| [keyframe-block-no-duplicate-selectors](https://stylelint.io/user-guide/rules/keyframe-block-no-duplicate-selectors/)                 | noDuplicateSelectorsKeyframeBlock |                                                                                          |
| [block-no-empty](https://stylelint.io/user-guide/rules/block-no-empty/)                                                               | noEmptyBlock                          |                                                                           |
| [keyframe-declaration-no-important](https://stylelint.io/user-guide/rules/keyframe-declaration-no-important/)                         | noImportantInKeyframe             |                                                                                          |
| [function-linear-gradient-no-nonstandard-direction](https://stylelint.io/user-guide/rules/function-linear-gradient-no-nonstandard-direction/) | noInvalidDirectionInLinearGradient |                                                                                          |
| [no-invalid-position-at-import-rule](https://stylelint.io/user-guide/rules/no-invalid-position-at-import-rule/)                       | noInvalidPositionAtImportRule     |                                                                                          |
| [no-irregular-whitespace](https://stylelint.io/user-guide/rules/no-irregular-whitespace/) |noIrregularWhitespace |  | 
| [declaration-block-no-shorthand-property-overrides](https://stylelint.io/user-guide/rules/declaration-block-no-shorthand-property-overrides/) | noShorthandPropertyOverrides      |                                                                                          |
| [function-no-unknown](https://stylelint.io/user-guide/rules/function-no-unknown/)                                                     | noUnknownFunction                 |                                                                                          |
| [media-feature-name-no-unknown](https://stylelint.io/user-guide/rules/media-feature-name-no-unknown/) | noUnknownMediaFeatureName |
| [property-no-unknown](https://stylelint.io/user-guide/rules/property-no-unknown/)                                                     | noUnknownProperty                 |                                                                                                                                                                          |
| [unit-no-unknown](https://stylelint.io/user-guide/rules/unit-no-unknown/)                                                             | noUnknownUnit                     |                                                                                          |
| [selector-anb-no-unmatchable](https://stylelint.io/user-guide/rules/selector-anb-no-unmatchable/)                                     | noUnmatchableAnbSelector          |                                                                                          |
| [named-grid-areas-no-invalid](https://stylelint.io/user-guide/rules/named-grid-areas-no-invalid/)                                     | useConsistentGridAreas            |                                                                                          |
| [font-family-no-missing-generic-family-keyword](https://stylelint.io/user-guide/rules/font-family-no-missing-generic-family-keyword/) | useGenericFontNames               |

**Comment by Conaclos** [comment]
@togami2864 Could you suggest a group for each one? (You can edit your comment).

**Comment by nhedger** [comment]
> A couple of other things for the website:
> 
> * update the language support page with graphql
> * documentation for assists (or I can do it when I come back)
> * documentation for VSCode v3
> * document the guide to explain that with `--verbose` is now possible to see all fixed files

VS Code v3 might not be ready by release time. I need more time to squash bugs.

**Comment by Conaclos** [comment]
@togami2864 
I choose a group for most of the rule.

If I am correct, `useConsistentGridAreas` is a correctness rule? However, its name sounds like a style rule. What do you think of renaming the rule to `noInvalidGridAreas`?

Also, Ia m thinking to `accessibility` for `useGenericFontNames` and `correctness` for all `noUnknown*` rules.

**Comment by togami2864** [comment]
@Conaclos 
> If I am correct, useConsistentGridAreas is a correctness rule? However, its name sounds like a style rule. What do you think of renaming the rule to `noInvalidGridAreas?`

Totally agree 👍 I'll make a PR.

> Also, Ia m thinking to accessibility for `useGenericFontNames`

This rule is originally from [font-family-no-missing-generic-family-keyword](https://stylelint.io/user-guide/rules/font-family-no-missing-generic-family-keyword/) in stylelint, I think it would be better to rename `noMissingGenericFamilyKeyword` and  assign `correctness` group. wdyt 🤔 

> We will delay the promotion of noIrregularWhitespace to a next release because, both the JS and CSS rules have to be promoted.
 
👍 

---

Besides that, I'll rename these rules to align with name conventions.

`no_unknown_pseudo_class_selector` => `no_unknown_pseudo_class`
`no_unknown_selector_pseudo_element` => `no_unknown_pseudo_element`

**Comment by Conaclos** [comment]
> This rule is originally from [font-family-no-missing-generic-family-keyword](https://stylelint.io/user-guide/rules/font-family-no-missing-generic-family-keyword/) in stylelint, I think it would be better to rename noMissingGenericFamilyKeyword and assign correctness group. wdyt 🤔

Why should this be a correctness rule? This is not fundamentally wrong / an error of omitting a generic font family?
Regarding the name, I am not sure if it makes so much value of renaming it.

**Comment by togami2864** [comment]
> This is not fundamentally wrong / an error of omitting a generic font family?

That makes sense. Let's keep the rule name and assign it as `accessibility`.

File: CHANGELOG.md [hunk 1]
```diff
@@ -285,6 +285,62 @@ our [guidelines for writing a good changelog entry](https://github.com/biomejs/b
...
+- [suspicious/noConsole](https://biomejs.dev/linter/rules/no-console/)
+- [suspicious/noEvolvingTypes](https://biomejs.dev/linter/rules/no-evolving-types/)
+- [suspicious/noMisplacedAssertion](https://biomejs.dev/linter/rules/no-misplaced-assertion/)
+- [suspicious/noReactSpecificProps](https://biomejs.dev/linter/rules/no-react-specific-props/)
+- [suspicious/useErrorMessage](https://biomejs.dev/linter/rules/use-error-message/)
+- [suspicious/useNumberToFixedDigitsArgument](https://biomejs.dev/linter/rules/use-number-to-fixed-digits-argument/)
+
+#### Deprecated rules
+
+- `style/noConsoleLog` is deprecated. Use [correctness/noConsole](https://biomejs.dev/linter/rules/no-console/) instead.
```
Comment (wojtekmaj) [inline CHANGELOG.md:340]: I think you've meant suspicious/noConsoleLog?

**Pull request merged**